config/Dockerfiles/linchpin/conf/ansible.cfg

```diff
@@ -0,0 +1,5 @@
+[defaults]
```

Comment:
I'd rather see this read from the workdir or at least allow the user to override it there.  I may want key checking

Reply:
@p3ck - To be fair, I don't think you would, as that would require you to be logged into the container while SSH'ing.

Reply:
Environment variables could be useful here also. We can obviously discuss this further.